content/pt-br/docs/concepts/security/pod-security-policy.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Políticas de Segurança do Pod
+content_type: conceito
+weight: 30
+---
+
+<!-- visão geral -->
+
+{{% alert title="Funcionalidade removida" color="warning" %}}
+PodSecurityPolicy foi [descontinuada](/blog/2021/04/08/kubernetes-1-21-release-announcement/#podsecuritypolicy-deprecation)
```

Comment:
Minor: little inconsistency between the chosen gender for PodSecurityPolicy. Here, you used "descontinuada", but in the next sentence, you used "removido". Update one of them to maintain consistency.